src/settings-ui/Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -17,133 +17,133 @@
+                                    Margin="56,0,0,8"/>
+                                <ToggleSwitch IsOn="{x:Bind IsShown, Mode=TwoWay}"
+                                                OnContent=""
+                                                OffContent=""
+                                                Grid.RowSpan="2"
+                                                x:Uid="Enable_ColorFormat"
+                                                AutomationProperties.HelpText="{x:Bind Name}"
+                                                HorizontalAlignment="Right"
+                                                Margin="0,0,56,0"/>
 
-                                    <Button Background="Transparent"
-                                            FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                            HorizontalAlignment="Right"
-                                            VerticalAlignment="Center"
-                                            Grid.RowSpan="2"
-                                            x:Uid="More_Options_Button"
-                                            Content="&#xEDE3;">
-                                        <Button.Flyout>
-                                            <MenuFlyout>
-                                                <MenuFlyoutItem IsEnabled="{x:Bind CanMoveUp}" Icon="Up" x:Uid="MoveUp" Click="ReorderButtonUp_Click"/>
-                                                <MenuFlyoutItem IsEnabled="{x:Bind CanMoveDown}" x:Uid="MoveDown" Click="ReorderButtonDown_Click">
-                                                    <MenuFlyoutItem.Icon>
-                                                        <FontIcon Glyph="&#xF0AE;" />
-                                                    </MenuFlyoutItem.Icon>
-                                                </MenuFlyoutItem>
-                                            </MenuFlyout>
-                                        </Button.Flyout>
-                                        <ToolTipService.ToolTip>
-                                            <TextBlock x:Uid="More_Options_ButtonTooltip" />
-                                        </ToolTipService.ToolTip>
-                                    </Button>
-                                </Grid>
-                            </DataTemplate>
-                        </ListView.ItemTemplate>
-                    </ListView>
-                </controls:SettingsGroup>
-            </StackPanel>
+                                <Button Background="Transparent"
+                                        FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                        HorizontalAlignment="Right"
+                                        VerticalAlignment="Center"
+                                        Grid.RowSpan="2"
+                                        x:Uid="More_Options_Button"
+                                        Content="&#xEDE3;">
```

Comment:
![image](https://user-images.githubusercontent.com/9866362/188415121-5858cbc5-637f-4786-8ba4-9fc90f41af9c.png)

We had the '...' before right? This is the Xbox controller menu button..

Reply:
True.